xaynetwork/xayn_discovery_app
# TB-4117 Fix Android app link verification

### What 🕵️ 🔍


When downloading the app from the play store, in order to open it when clicking on a deep link, it seems that we first have to manually add the link to the "links to open the app" section in the settings of the device.

AppSflyer's customer support suggested to move the default scheme out of the intent filter that uses the auto:verify

----------

### How to test (please adjust template) 🥼 🔬

This can only be tested with a build that has the same SHA fingerprint used in the play console

- Download the app
- On your Android device, go to settings->default apps->opening links
- Find the app and check that  `news-app.onelink.me` is already added to the supported links list of the app

Please contact me when testing the PR. Will give you all the needed info.
 

----------

### Screenshots 📸 📱

| Before | After  |
| ------ | ------ |
| <img width="280" src=""> | <img width="280" src=""> |

----------

### References 📝 🔗

- [JIRA](https://xainag.atlassian.net/browse/TB-4117)

----------
